markdown/developers/sync.md

```diff
@@ -69,7 +69,7 @@ Next, check out a new branch to make these changes in:
 git checkout -b merging-template-updates
 ```
 
-Finally, pull the `TEMPLATE` branch from the `upstream` repo:
+Finally, pull the `TEMPLATE` (or `nf-core-template-merge-<version>`) branch from the `upstream` repo:
```

Comment:
No need for this change, I think it just complicates matters when pulling to a fork - the two should be identical.